src/core/style/_variables.scss

```diff
@@ -20,6 +20,9 @@ $md-toggle-padding: 8px !default;
 // Width and height of input toggles
 $md-toggle-size: 20px !default;
 
+
+$md-input-floating-placeholder-scale-factor: 0.75 !default;
```

Comment:
Variables that are specific to a single component should live in that component's SCSS.

Just move this to the file where it's used.